src/test/java/com/aws/greengrass/certificatemanager/certificate/CertificateHelperTest.java

```diff
@@ -36,7 +36,7 @@ public class CertificateHelperTest {
     private static final String EXPECTED_ISSUER_PRINCIPAL
             = "CN=Greengrass Core CA,L=Seattle,ST=Washington,OU=Amazon Web Services,O=Amazon.com Inc.,C=US";
     private static final String EXPECTED_SUBJECT_PRINCIPAL
-            = "CN=testCNC\\=USST\\=WashingtonL\\=SeattleO\\=Amazon.com Inc.OU\\=Amazon Web Services";
+            = "CN=testCN,C=US,ST=Washington,L=Seattle,O=Amazon.com Inc.,OU=Amazon Web Services";
```

Comment:
how could we have validated this before, using Java? Why are we only finding this now?

Reply:
Good question. I don't think anything really looks at this.

Theoretically we could add a test that parses the Subject and pulls out the individual fields. Without this change we just get a string with no semantic meaning.